ops/services/alerts/app_service_metrics/main.tf

```diff
@@ -281,22 +281,22 @@ resource "azurerm_monitor_scheduled_query_rules_alert" "batched_uploader_single_
   resource_group_name = var.rg_name
   severity            = var.severity
   frequency           = 5
-  time_window         = 7
+  time_window         = 10
   enabled             = contains(var.disabled_alerts, "batched_uploader_single_failure_detected") ? false : true
 
   data_source_id = var.app_insights_id
 
   query = <<-QUERY
 requests
 ${local.skip_on_weekends}
-| where timestamp >= ago(7m) 
+| where timestamp >= ago(10m) 
```

Comment:
Should we maybe make this 11m, just to account for any weirdness like  it being just a second off?

Reply:
Yes we should, good idea.